win32ss/user/rtl/image.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * PROJECT:   ReactOS RTL
+ * LICENSE:   LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:   RtlGetExpWinVer function
+ * COPYRIGHT: Copyright 2024 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#ifdef _WIN32K_
+    #include <win32k.h>
+
+    #define NDEBUG
+    #include <debug.h>
+#else
+    #include <user32.h>
+    #include <wine/debug.h>
+
+    WINE_DEFAULT_DEBUG_CHANNEL(user32);
+#endif
```

Comment:
Please do something more closely similar to [this](https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/user/rtl/text.c;h=ac12626796b3d8b9d6359a87155deeec7fe64a59;hb=HEAD#l29):
```
#ifdef _WIN32K_
#include <win32k.h>
DBG_DEFAULT_CHANNEL(UserMenu);
#else
#include <user32.h>
#include <wine/debug.h>
WINE_DEFAULT_DEBUG_CHANNEL(text);
#endif
```
- indentation not needed;
- the "NDEBUG"/debug.h in the win32k case not wanted, please use `DBG_DEFAULT_CHANNEL(UserMisc);` or `DBG_DEFAULT_CHANNEL(UserProcess);`
- in the non-win32k case (the `#else` block), use `WINE_DEFAULT_DEBUG_CHANNEL(user32);`

Reply:
```suggestion
#ifdef _WIN32K_
#include <win32k.h>
DBG_DEFAULT_CHANNEL(UserMisc);
#else
#include <user32.h>
#include <wine/debug.h>
WINE_DEFAULT_DEBUG_CHANNEL(user32);
#endif
```